README.md

```
@@ -1,3 +1,52 @@
 # java-janggi
 
-장기 미션 저장소
+## 장기 미션
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)